feature-group-definitions/navigation-api.yml

```diff
@@ -0,0 +1 @@
+spec: https://wicg.github.io/navigation-api/
```

Comment:
This and Sanitizer have boxed us in with one-word-and-API titles. This is probably fine.

Reply:
I think it is fine if we have to use -api to disambiguate, but just `navigation` here actually seems reasonable. There's a `window.navigation` property which can be used like `navigation.navigate(url)`. So I think I'll drop the suffix actually.